bakery/env/prod.json

```diff
@@ -15,6 +15,7 @@
   "GDOC_GOOGLE_FOLDER_ID": "",
   "GDOC_QUEUE_FILENAME": "",
   "GDOC_FEED_FILE_URL": "",
+  "GH_SECRET_CREDS": "((github-username)):((github-api-token))",
```

Comment:
More of a question from curiosity than a code comment, but did you find the username being necessary here? When running the CLI I've just been using my access token string for `gh-creds` w/o username and it worked okay 🤷‍♂️ .

Reply:
Hmm.... I never even thought to test it without a username!